### PR TITLE
zh: modify format of `--advertise-peer-urls`

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

modify format of `--advertise-peer-urls` to align with https://github.com/pingcap/dm/pull/955

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

**Review by ireneontheway — COMMENTED** [review]
LGTM @yikeke

**Comment by yikeke** [comment]
master 分支也得提个 pr 改一下 @lance6716 

文档这边一般的流程是先提 PR 到 master 分支，然后打 cherry-pick 的标签，可以方便地触发 bot 自动 cherry-pick 到你想要的 release 分支，FYI @lance6716

File: zh/command-line-flags.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/command-line-flags/']
 
 - DM-master 节点间通信的外部连接地址
 - 默认值为 `"{peer-urls}"`
-- 可选参数，可以为 `"域名:port"` 的形式
+- 可选参数，可以为 `"http(s)://域名:port"` 的形式
```
Comment (yikeke) [inline zh/command-line-flags.md:23]: 改动很小的话，en/command-line-flags.md 也一起改下？ @lance6716

Comment (lance6716) [reply zh/command-line-flags.md:23]: 我新开一个到master的，同时修改en

Comment (yikeke) [reply zh/command-line-flags.md:23]: 好的，多谢

**Pull request closed without merging**